yaml/gateway-dep.yml

```diff
@@ -59,6 +59,13 @@ spec:
       - name: faas-netes
         image: openfaas/faas-netes:0.6.0
         imagePullPolicy: Always
+        livenessProbe:
```

Comment:
This would have to be added to the helm chart also.

Reply:
done